tests/ballerina-unit-test/src/test/resources/test-src/jvm/types.bal

```diff
@@ -213,3 +213,27 @@ function acceptAnydata(anydata data) returns anydata {
    ia[1] = 1000;
    return ia[1];
 }
+
+int global = 0;
```

Comment:
I think its better to move future tests to separate bal.

Reply:
Why? Future also a type.

Reply:
May be once we have other operations with futures better to move to new file. +1 for now to use same types.bal.